server/controllers/utils/formatReviews.js

```diff
@@ -64,9 +58,11 @@ const format = review => {
 
   // magical factors
   copy.magicalFactors = keys
-    .filter(e => e.split('_')[0] === 'mf')
+    .filter(e => {
+      const key = e.split('_')[0];
+      return key === 'mf' || key === 'bc';
```

Comment:
do not hardcode string literals that are used for logic, use enums/constants

Reply:
these are coming from Airtable, that's a quick trick, its actually more flexible than u think XD
you can see the fields in the stay, eat, shop: https://airtable.com/shrWi4h598CkxhYlV

Reply:
will delete link soon

Reply:
my point was that you should put `'mf'` and `'bc'` into an enum/constant and not hardcode it. Also, to improve readability, it would be good to put it into a function `isMagicalFactor()`, so it would look like `.filter(e => isMagicalFactor(e))`

As for the solution, it would be worth considering having just one field in the database containing all of the magical factors in a string/array (e.g. `local_immersion staff_interaction`), instead of having a separate field for each magical factor. It would be more flexible - adding or removing new magical factors wouldn't require changing the DB schema and here you wouldn't have to do these shenanigans with the object's keys.

Reply:
there is a field named: `magical factors` but its not 'keyed', and id be using string and maybe emojis to compare strings, but the thing is in the typeform, when the user picks a magical factor, he is asked to fill another single line text field, which are the ones prefixed with `mf` or `bc`

Reply:
ok I'm obviously missing information about the whole thing so can't really comment on that 🙂

Reply:
but that comment about not hardcoding strings stands 😉